addons/web/static/src/search/group_by_menu/group_by_menu.js

```diff
@@ -69,4 +69,5 @@ GroupByMenu.components = { CustomGroupByItem };
 GroupByMenu.template = "web.GroupByMenu";
 GroupByMenu.defaultProps = {
     showActiveItems: true,
+    showCaretDown: false,
```

Comment:
Before this commit, the menu "Group By" of a sub graph view in a dahsboard view was not styled the right way:
it did't have the symbol "fa-caret-down" and did look visually different from the menu "Measures".
Here we define a new prop "showCaretDown" for the class GroupByMenu to control the apparition of "fa-caret-down" and use it when suitable.